src/components/Filter/LessonFilter.js

```diff
@@ -18,11 +19,25 @@ export const LessonFilter = React.createClass({
     const title = <h3>Filter</h3>;
     const bsStyle = (this.props.isStudentMode ? 'student' : 'teacher');
     return (
-      <Panel header={title} bsStyle={bsStyle}>
-        {filterGroups}
-        <br/>
-        <Button block onClick={() => this.props.resetFilter()}>Fjern filter</Button>
-      </Panel>
+      <div>  
```

Comment:
Forøvrig, som det står her, så er det bedre å ha popover/tooltip ved click enn hover, pga universell utforming (accessibility), altså folk som brukere screenreaders (svaksynte/blinde) m.m.:
https://react-bootstrap.github.io/components.html#popovers-trigger-behaviors

La også på en rootClose sånn at den lukker seg hvis man klikker utenfor (ikke bare på knappen).